low-level/files/opt/seagate/sspl/conf/sspl.conf.LR2.yaml

```diff
@@ -71,8 +71,8 @@ LOGGINGMSGHANDLER:
 
 NODEDATAMSGHANDLER:
    transmit_interval: 10
-   high_cpu_usage_wait_threshold: 30
-   high_memory_usage_wait_threshold: 30
+   high_cpu_usage_wait_threshold: 10
```

Comment:
I'm assuming our test code will wait for 10 seconds before checking CPU usage again? Is that right?

Reply:
@thavanathan the cpu/memory sensors raises alert only if usage remains over the threshold for the defined time period.
Earlier these wait threshold frequency was 30sec which was high and beyond test limit,
So I reduced the time accordingly and now alerts can be seen in this time period